quadkey/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,148 @@
+## Reference
+
+### QUADKEY
+
+#### quadkey.VERSION
+
+{{% bannerNote type="code" %}}
+quadkey.VERSION () -> INT64
+{{%/ bannerNote %}}
+
+Returns the current version of the quadkey library.
+
+#### quadkey.QUADINT_FROMZXY
+
+{{% bannerNote type="code" %}}
+quadkey.QUADINT_FROMZXY(z INT64, x INT64, y INT64) -> INT64
+{{%/ bannerNote %}}
+
+* `z`: `INT64` Level of zoom.
+* `x`: `INT64` horizontal position of a tile.
+* `y`: `INT64` vertical position of a tile.
+
+Returns the quadint representation for tile x, y and a zoom z.
+
+#### quadkey.ZXY_FROMQUADINT
+
+{{% bannerNote type="code" %}}
+quadkey.ZXY_FROMQUADINT(quadint INT64) -> STRUCT< z INT64, x INT64, y INT64 >
+{{%/ bannerNote %}}
+
+* `quadint`: `INT64` quadint we want to extract tile information from.
+
+Returns the level of zoom z and coordinates x, y for a given quadint.
+
+#### quadkey.LONGLAT_ASQUADINT
+
+{{% bannerNote type="code" %}}
+quadkey.LONGLAT_ASQUADINT(longitude FLOAT64, latitude FLOAT64, resolution INT64) -> INT64
+{{%/ bannerNote %}}
+
+* `longitude`: `FLOAT64` horizontal coordinate of the map.
+* `latitude`: `FLOAT64` vertical coordinate of the map.
+* `resolution`: `INT64` Level of detail or zoom.
+
+Returns the quadint representation for a given level of detail and geographic coordinates.
+
+#### quadkey.QUADINT_FROMQUADKEY
+
+{{% bannerNote type="code" %}}
+quadkey.QUADINT_FROMQUADKEY(quadkey STRING) -> INT64
+{{%/ bannerNote %}}
+
+* `quadkey`: `STRING` quadkey we want to convert to quadint.
+
+Transform a quadkey index to an equivalent quadint.
+
+#### quadkey.QUADKEY_FROMQUADINT
+
+{{% bannerNote type="code" %}}
+quadkey.QUADKEY_FROMQUADINT(quadint INT64) -> STRING
+{{%/ bannerNote %}}
+
+* `quadint`: `INT64` quadint we want to convert to quadkey.
+
+Transform a quadint index to an equivalent quadkey.
+
+#### quadkey.PARENT
+
+{{% bannerNote type="code" %}}
+quadkey.PARENT(quadint INT64) -> INT64
+{{%/ bannerNote %}}
+
+* `quadint`: `INT64` quadint we want to get the parent from.
+
+Returns the parent quadint of a given quadint. A parent quadint is a quadint of smaller level of detail (zoom - 1) which contains the current quadint.
+
+#### quadkey.CHILDREN
```

Comment:
Same, in `h3` this function is called `TOCHILDREN` and supports passing the resolution of the children you want.

Reply:
This should be also doable. I guess this should return all the children contained for a given resolution right? Example, current quadint is in resolution 1, TOCHILDREN(quadint, 3)-> array with 16 children.